src/node.rs

```diff
@@ -87,20 +86,6 @@ impl Node {
     }
 }
 
-impl Spanned for Node {
```

Comment:
From what I could read ToTokens implements that now from what I could see, since there is little information on how to upgrade from syn 1 to syn 2. But I can try again and see what may be needed for a manual Spanned implementation, since when I tried to impl the required traits required by Spanned it was a little overwhelming.

Reply:
yes, `Spanned` is now marked as `Sealed`, also `Spanned` from quote lib also marksed as `Sealed` and has auto impl from ToToken.
ToToken implementation is possible with quote for current structs, but i am not sure if quote will not mess with spans. I think https://github.com/stoically/syn-rsx/issues/53 can be related.

Reply:
I also pushed [a branch to my fork](https://github.com/stoically/syn-rsx/compare/main...vldm:update-syn?expand=1) that refactor syn-rsx a bit, and contain ToToken implementation. It contain a lot of changes in file structure, so i am sure that it wont be merged =(
But there [my version of syn update](https://github.com/stoically/syn-rsx/commit/e65938b705072918f7d665a6dd08c5fc1fca8e0b), and you can take ToToken impl from it, if you need. 
There also block_transform impl, that can be useful.

Reply:
Cool, I will have a look. I already pushed some changes and right now the main lingering issue is the ToTokens, which I'm trying to figure out what make sense to send to the token stream, so for sure your implementation will help.

Reply:
I used your ToTokens implementation and they make sense, at least to me.